masonry/src/core/contexts.rs

```diff
@@ -594,12 +594,8 @@ impl LayoutCtx<'_> {
         self.get_child_state(child).baseline_offset
     }
 
-    /// Get the given child's layout rect.
-    ///
-    /// ## Panics
-    ///
-    /// This method will panic if [`LayoutCtx::run_layout`] and [`LayoutCtx::place_child`]
-    /// have not been called yet for the child.
+    // TODO - Remove (used in Flex)
+    #[doc(hidden)]
     #[track_caller]
     pub fn child_layout_rect(&self, child: &WidgetPod<impl Widget + ?Sized>) -> Rect {
```

Comment:
If it's only used in Flex maybe:
```suggestion
    pub(crate) fn child_layout_rect(&self, child: &WidgetPod<impl Widget + ?Sized>) -> Rect {
```

Might also be worth to fix the parts where the code is used in `Flex`?
So that we can get rid of it entirely here?

Reply:
I think that's a whole PR on its own.